docker-compose.yml

```diff
@@ -12,13 +14,18 @@ services:
       dockerfile: Dockerfile.dev
     volumes:
       - "./src:/app/src"
-    depends_on:
-      - redis
-    links:
-      - redis
+    networks:
+      - frontend
+      - backend
     environment:
       NODE_ENV: development
       HRM_HOST: "0.0.0.0"
     ports:
       - "3002:3002"
       - "8081:8081"
+
+networks:
```

Comment:
why do we need two networks?

Reply:
As far as I understand, yes, it will be better to have two networks in order to separate different types of containers.